tests/plugin_integration.suite.dist.yml

```
@@ -0,0 +1,36 @@
+actor: Plugin_integrationTester
+bootstrap: _bootstrap.php
+modules:
+    enabled:
+        - \Helper\Plugin_integration
+        - WPLoader
+        - WPDb
+    config:
+      WPDb:
+        dsn: 'mysql:host=%WP_DB_HOST%;dbname=%WP_DB_NAME%'
+        user: %WP_DB_USER%
+        password: %WP_DB_PASSWORD%
+        dump: tests/_data/plugin_integration/dump.sql
+        populate: true # Do import the dump when the suite starts.
+        cleanup: false # Do not clean the db between tests, we'll deal with it.
+        url: '%WP_URL%'
+        originalUrl: 'http://wordpress.test'
+        tablePrefix: '%WP_TABLE_PREFIX%'
+        waitlock: 10
+      WPLoader:
```

You do not need all the configuration parameters when using the `loadOnly: true` mode.
[See here](https://wpbrowser.wptestkit.dev/modules/wploader#wploader-to-only-bootstrap-wordpress).

```suggestion
      WPLoader:
        loadOnly: true
        wpRootFolder: "%WP_ROOT_FOLDER%"
        dbName: "%WP_TEST_DB_NAME%"
        dbHost: "%WP_TEST_DB_HOST%"
        dbUser: "%WP_TEST_DB_USER%"
        dbPassword: "%WP_TEST_DB_PASSWORD%"
        domain: "%WP_DOMAIN%"
```

The plugins are activated in the dump and do not need specification.